ibm/mas_devops/roles/kubeturbo/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Define where Kubeturbo will be installed, and to which Turbonomics it will connect to
+turbonomic_route: "{{ lookup('env', 'TURBONOMIC_ROUTE') }}"
+turbonomic_password: "{{ lookup('env', 'TURBONOMIC_PASSWORD') }}"
+turbonomic_username: "{{ lookup('env', 'TURBONOMIC_USERNAME') }}"
+kubeturbo_namespace: "{{ lookup('env', 'KUBETURBO_NAMESPACE') | default('kubeturbo', true) }}"
+kubeturbo_sub_name: "{{ lookup('env', 'KUBETURBO_SUBNAME') | default('kubeturbo-certified', true) }}"
```

Comment:
is there a reason why we might have a different subscription name? Normally we just make the channel configurable rather than the subscription name itself.

Reply:
In https://github.com/ibm-mas/ansible-devops/pull/792/files#diff-0c82412b424decc5104495be47b13c23855569c028e5b10c845bf581b3434f90 just set a specific name .. no need to make it configurable, we don't anywhere else.

e.g.

```
name: "{{ kubeturbo_sub_name }}"
```

to

```
name: kubeturbo
```